### PR TITLE
Task7-Fetching and Displaying Data

### PR DESCRIPTION
**Pull request by mishalfirdous**

### CODE REVIEW

File: getData.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      tab += `<tr>
+                <td>${user.firstName}</td>
+                <td>${user.lastName}</td>
+                <td>${user.age}</td>
+                <td>${user.email}</td>
+                <td>${user.gender}</td>
+                </tr>`;
+    })
+    document.getElementById('tbody').innerHTML = tab;
+    //console.log(getData());
```
Comment (aaminbaig) [inline getData.js]: Please avoid submitting commented code and console logs..

File: signup.js [hunk 2]
```diff
@@ -73,6 +76,7 @@ document.addEventListener("DOMContentLoaded", function () {
     if (response.status === 200) {
       registrationStatus.textContent = "Registration successful";
       registrationStatus.style.display = "block";
+      console.log(userData);
```
Comment (aaminbaig) [inline signup.js]: Please remove console logs.

File: stylesheet.css [hunk 3]
```diff
@@ -218,6 +219,7 @@ table tr {
   border: 1px solid #ccc;
 }
 form {
+  margin-top: 0px;
   margin-top: 10px;
```
Comment (aaminbaig) [inline stylesheet.css]: You have added 2 margin-top properties.

**Review by aaminbaig — CHANGES_REQUESTED** [review]

File: getData.js (hunk 1 above)
Comment (mishalfirdous) [reply getData.js]: fixed!

File: signup.js (hunk 2 above)
Comment (mishalfirdous) [reply signup.js]: fixed!

File: stylesheet.css (hunk 3 above)
Comment (mishalfirdous) [reply stylesheet.css]: fixed!

**Review by aaminbaig — APPROVED** [review]

**Pull request merged**